code/modules/surgery/organs/parasites.dm

```diff
@@ -0,0 +1,35 @@
+/obj/item/organ/internal/body_egg/spider_eggs
+	name = "spider eggs"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "eggs"
+	var/stage = 1
+
+/obj/item/organ/internal/body_egg/spider_eggs/on_life()
+	if(stage < 5 && prob(3))
```

Comment:
I think the convention is that the egg's stage is advanced in `egg_process()` not `on_life()`.